app/src/main/java/com/yeonae/chamelezone/view/course/CourseModifyActivity.kt

```diff
@@ -51,25 +53,26 @@ class CourseModifyActivity : AppCompatActivity(), CourseModifyContract.View,
     }
 
     override fun showCourseDetail(courseList: List<CourseResponse>) {
+        originCourseList = courseList
         savedImageName = courseList[0].courseImage
         firstPlaceNumber = courseList[0].placeNumber
         secondPlaceNumber = courseList[1].placeNumber
         imageNumber = courseList[0].courseImageNumber
         edt_course_title.text = SpannableStringBuilder(courseList[0].title)
         edt_course_content.text = SpannableStringBuilder(courseList[0].content)
 
-        imageContainer.removeAllViews()
-        val rlSlideImg = LayoutInflater.from(this).inflate(
-            R.layout.slider_item_image,
-            imageContainer,
+        image_container.removeAllViews()
+        val clSliderImg = LayoutInflater.from(this).inflate(
+            R.layout.slider_course_image,
+            image_container,
             false
-        ) as RelativeLayout
-        imageContainer.addView(rlSlideImg)
-        rlSlideImg.findViewById<ImageView>(R.id.image_item).run {
+        ) as ConstraintLayout
```

Comment:
```suggestion
        ) as ViewGroup
```
ConstraintLayout의 속성을 사용할 것이 아니라면 부모클래스인 ViewGroup을 사용하는 것이 위같은 경우(RelativeLayout -> ConstraintLayout) 같은 변경사항에 유연할 것 같아요

Reply:
수정했습니당
9710f94